### DISCUSSION TITLE
how to contribute?

### DISCUSSION THREAD

**Discussion by FilipePfluck** [cschroeter/park-ui#33] [General]
hey @cschroeter . This is not an actual issue, just a question. I want to get in touch with you. 
For the last few months, I've been doing something extremelly simillar to park-ui, without even knowing about your project: 
https://github.com/FilipePfluck/PlucoUI

So I want to start contributing to park-ui instead. Is there anything you want my help with? Is there any way to communicate better with you? Perhaps you could create a discord community for park ui? Also I noticed this project lacks a Contributing.md or a great Readme.md, so I just don't have any idea on how to get started. Perhaps you could instruct me further? Thanks.

**Comment by coopbri** [comment]
+1 on a Discord community for Park UI (or perhaps a channel in e.g. the Chakra or Panda servers)!

**Comment by cschroeter** [comment]
For now if you want to dedicate some time to this project please reach out on me on Twitter.

**Reply by ousszizou** [reply]
I sent u a msg on Twitter
